cmd/build.go

```diff
@@ -284,6 +287,7 @@ func dobuild(params buildParams, args []string) error {
 		WithOptimizationLevel(params.optimizationLevel).
 		WithOutput(buf).
 		WithEntrypoints(params.entrypoints.v...).
+		WithRegoAnnotationEntrypoints(true).
```

Comment:
:+1: This is a decision I'd agree with: by default, entrypoint metadata will be effective. There's no way to turn it off, but you could just not use it. Since it's not been an allowed keyword before (for metadata), no existing policy should be affected by this.